### ISSUE TITLE
The Output Inspector is wrong to show in this case?

### ISSUE THREAD

**Issue by martinduo** [wallabyjs/public#2706]
The Output Inspector is wrong to show in this case?

In this situation I have only just opened my VS Code project. No application code has been executed. After some seconds, the Output Inspector pops up with this window.

![image](https://user-images.githubusercontent.com/879852/118363494-910e7b80-b562-11eb-8d57-35b1756df35d.png)

**Clicking "Debug test" does nothing. Clicking "Go to test" takes me to this Jest test file.**

![image](https://user-images.githubusercontent.com/879852/118363686-915b4680-b563-11eb-81ae-1ef53bb55e8c.png)

Since no app or test code has been run, it makes no sense to report a problem with this test, or the line of code shown in the Output Inspector. The "zz" variable represents an object that has not been initialized and will not have the properties used in the test.

And why are there so many repeated lines of the same problem code in the Output Inspector?

### Wallaby diagnostics report

module.exports = () => {
  return {
    files: ['src/**/*.js'],
    tests: ['src/**/*.spec.js'],
    env: {
      type: 'node',
    },
  }
}

**Comment by NikGovorov** [comment]
Hi @martinduo, 

Could you please share the full content of line 2097 in `moduleCalc2.js`? I assume that you have [Object proxy](https://wallabyjs.com/docs/intro/advanced-logging.html#object-proxy) live comment at the end of the line with an expression that can not be handled.

>And why are there so many repeated lines of the same problem code in the Output Inspector?

Apparently that line is executed many times during `submits plan for object calculation` test execution (perhaps in `beforeEach`).

**Comment by martinduo** [comment]
**This is the full line 2097 of moduleCalc2 from a copy/paste:**

        zz.planOverallResults.postResultRecord(currentYear, 0, acctObject.yearsIncome, acctObject.yearsTaxPayments + acctObject.yearsNetPayout, acctObject.yearsNetPayout, 0)

**Here is a screenshot showing what is visible at the end in addition to the file's text.**
![image](https://user-images.githubusercontent.com/879852/118482554-e2457900-b6e2-11eb-9e48-6c748a2a1297.png)

**And here is the only Jest test that uses that code:**

`describe('With moduleCalc0', () => {
  beforeEach(() => {
    let planObject = cloneDeep(samplePlan) // get the parsed object from the imported json file
    hydratePlanObject(planObject) // build helper functions and dictionaries, call createPlanObjectReferences
    gg.jestRunning = true // stop some runCatcher action & bypass some debugger statements
  })

  afterEach(() => {
    gg.jestRunning = false // allow runCatcher action & debugger statements
    gg.jestExpectingThrow = false // prevents userErrorMsg to throw
  })

  it('submits plan object for calculation', () => {
    expect(zz.accounts.count()).toBeGreaterThan(0)
    calcRequestObject.planObject = zz.planObject
    // expect(() => {
    calculationEntry(calcRequestObject)
    // }).not.toThrow()
  })
})
`

**Comment by NikGovorov** [comment]
> Is that in fact what the Output Inspector is telling me?
No, the Output Inspect should not show the messages. 

Could you please start Wallaby, edit `moduleCalc2.js` by adding a new line to the end of the file for example and see if the inline message at the end of line 2097 disappears? If it does not, please stop Wallaby and run `npx jest --clearCache `, then start Wallaby again, it should not show any messages on line 2097.

**Comment by martinduo** [comment]
Sorry, but I haven't had the chance to try your suggestion. I've had to
chop up mycode to try to fix a messy problem { a cyclic reference?). But
I'll get back to it soon.

**Comment by martinduo** [comment]
I'm back with running code again. I did your experiment but nothing
changed. I still have line 2097 looking like this:
[image: image.png]

**Comment by martinduo** [comment]
Hey, here is the cause of line 2097's strange output. I decided to watch
your latest video and saw about the Wallaby tools and paused the video to
experiment. Anyway, using the value inspector (or something, I have not
been aware of these tools) I saw the output from a log that referenced line
2097. So I did something to investigate (sorry that I'm not explaining all
this correctly) and saw this in my code:

2097 zz.planOverallResults.postResultRecord(current....
2098
2099 // ?? acctObject.postYearEndValues([acctObject...

That line with the ?? is a years old note to myself that this line is
questionable and should be looked at later when I'm in this area again. The
Wallaby output shows the source as  acctObject.postYearEndValues. If I
delete the ?? the stuff at the end of line 2097 goes away.

So this is something just triggered by the new Output Inspector. And I,
without any knowledge (yet) of how these tools work submitted a report.
